__tests__/app.test.js

```diff
@@ -327,6 +327,61 @@ describe('/api/users', () => {
 
 })
 
+describe('/api/reviews', () => {
+    test('GET-Status: 200 - responds with array of all users ', () => {
+        return request(app).get("/api/reviews").expect(200).then(({ body }) => {
+
+
+            expect(Array.isArray(body.reviews)).toBe(true)
+            expect(body.reviews.length).toBeGreaterThan(0);
+
+        })
+    })
+    test('GET-status: 200 order by descending by default', () => {
+        return request(app).get("/api/reviews").expect(200).then(({ body }) => {
+            expect(body.reviews).toBeSortedBy("created_at", {
+                descending: true
+            })
+
+        })
+    });
+    test('GET-status: 200 order by ascending if passed on query', () => {
+        return request(app).get("/api/reviews?order=asc").expect(200).then(({ body }) => {
+            expect(body.reviews).toBeSortedBy("created_at", {
+                ascending: true
+            })
+
+        })
+    });
+    test('GET-status: 200 order by descending if passed on query', () => {
+        return request(app).get("/api/reviews?order=desc").expect(200).then(({ body }) => {
+            expect(body.reviews).toBeSortedBy("created_at", {
+                descending: true
+            })
+
+        })
+    });
+
+    test('GET-status: 200 can be sort by valid colum specified to query', () => {
+        return request(app).get("/api/reviews?sort_by=votes").expect(200).then(({ body }) => {
+            expect(body.reviews).toBeSortedBy("votes", {
+                descending: true
+            })
+
+
+        })
+    });
+
+    test('GET-status:400 invalid sort criteria', () => {
+        return request(app).get("/api/reviews?sort_by=bananas").expect(400).then(({ body }) => {
+            expect(body.message).toBe("invalid sort query")
+
+        })
+    });
+
+
```

Comment:
So we'd just need a few more tests here please! 

Status 200, accepts category query, e.g. ?category=dexterity.
should check that all returned objects have the queried category.

Status 400. invalid order query, e.g. ?order=bananas

Status 200, valid category query, but has no reviews responds with an empty array of reviews, e.g. ?category=children's games. 

Status 404, non-existent category query, e.g. ?category=bananas.